README.md

```diff
@@ -169,6 +169,16 @@ sudo apt install openjdk-<version>-jdk
 
 E.g.: `sudo apt install openjdk-8-jdk`
 
+### termux (experimental)
+You must install java-17 and few more thing:
+
+```bash
+pkg install openjdk-17 aapt apksigner dx ecj
+```
+Download, unzip and set enviroment variable to sdk and ndk from https://github.com/Lzhiyong/termux-ndk/releases 
+you maybe will have to set sdk version into compatible with ndk (`sdkmanager install/uninstall platform-version`) 
```

Comment:
Keeping lines below 100 chars in markdown.md files will keep the CI checks happy:
```suggestion
you will maybe have to set the SDK version to be compatible with the NDK
(`sdkmanager install/uninstall platform-version`) 
```